### PR TITLE
Add acknowledgments

### PR DESCRIPTION
**Pull request by joan-ing**
Closes #

## Summary
* Add text for README

### CODE REVIEW

**Comment by joan-ing** [comment]
@CdrMarks @sugaroverflow Could you please review this PR?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+So, you want to get involved:
+
+- [Contributor's guide](https://github.com/pantheon-systems/gift-of-open-source/tree/master/content/contributors.md) - Start here if this is your first time here or need a refresher on how to get started. This document is for anyone who would like to learn how to contribute to the event and win great swag.
+
+- [Maintainer's guide](https://github.com/pantheon-systems/gift-of-open-source/tree/master/content/maintainers.md) - Review these instructions if you are a code maintaniner and would like to participate.
+
+
+## Engagement
+
+- Live streams Schedule
```
Comment (CdrMarks) [inline README.md]: Let's not mention this here until we have something substantive to go with it.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,44 @@
...
+
+
+## Engagement
+
+- Live streams Schedule 
+
+
+## Questions
+
+If you have any questions or concerns, you can send a message in the [Gift of Open Source](https://pantheon-community.slack.com/archives/C01D3VBLAHM) community Slack. You can also [submit an issue](https://github.com/pantheon-systems/gift-of-open-source/issues/new/choose) to this [repo](https://github.com/pantheon-systems/gift-of-open-source).
```
Comment (CdrMarks) [inline README.md]: Since they are reading this on GitHub, can you list submitting an issue first and the Slack Community second?

Comment (CdrMarks) [inline README.md]: Should the target text for the Slack Community be `#gift-of-open-source Slack channel`?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,44 @@
...
+
+* [Gift of Open Source](https://pantheon.io/gift-open-source)
+* [Registration](https://pantheon.io/gift-open-source#register)
+
+
+_____________________________
+
+## Acknowledgements
+
+We would like to give credit to Hacktoberfest 2021 for inspiring the text in this repository.
```
Comment (CdrMarks) [inline README.md]: How about this instead:
Shout out to Hacktoberfest for inspiring this project and creating a model for the Gift of Open Source.

**Review by CdrMarks — CHANGES_REQUESTED** [review]

**Pull request merged**